tox.ini

```diff
@@ -1,41 +1,33 @@
 [tox]
 downloadcache = {toxworkdir}/_download/
-envlist = py26-1.4.X,py26-1.3.X,py26-1.2.X,docs
+envlist =
+    {py26,py27}-{django1.6.X},
+    {py27}-{django1.7.X},
+    docs
 
 [testenv]
-commands = django-admin.py test decisiontree
+basepython =
+    py26: python2.6
+    py27: python2.7
+
+commands =
+    coverage run {envbindir}/django-admin.py test decisiontree
+    flake8
+    coverage report -m
```

Comment:
Another option would be to make a separate testenv for flake8 and coverage, so that those don't run for each build. (Flake8, at least, shouldn't change from environment to environment, but coverage possibly would)